src/cloud-cli/meltano/cloud/cli/project.py

```diff
@@ -288,3 +311,39 @@ async def use_project(
         ),
         fg="green",
     )
+
+
+@project_group.command("create")
+@click.option("--project-name", type=str, required=True)
+@click.option("--git-repository", type=str, required=True)
+@click.option("--project-root-path", type=str, required=False)
```

Comment:
Two suggestions here:
- Use `prompt=True` as we do for the deployments subcommand to prompt users if they fail to provide a required parameter. This makes this easier for interactive/human users, while not changing the behaviour in a non-interactive context.
- Simplify the parameter names. The `project- prefix is redundant since this is the `project` subcommand, and `git-repository` doesn't clearly indicate that it's expect the URL to the repository. 

```suggestion
@click.option("--name", type=str, prompt=True)
@click.option("--repo-url", type=str, prompt=True)
@click.option("--root-path", type=str, required=False)
```